### ISSUE TITLE
Please tell me the name of the growing plant.

### ISSUE THREAD

**Issue by ElseJunsuk** [SkriptLang/Skript#3511]
### Description
i can't pinpoint the name of the plant it's growing. This is a problem that has been resolved in the previous version 2.5.0.
To find out the name of the plant, i wroted a pharase to find the name of the 'targeted item' in on command. But the growing plants all had same name. please fixing it

### Errors / Screenshots
![plz](https://user-images.githubusercontent.com/53819958/97869617-22e83100-1d55-11eb-8444-7fdc06188104.PNG)
in the phot above. i have to put the name of the wheat grown to 100%  in the part marked "stage eight wheat plant", but i don't know the name. if you continue, you will recognize the names of wheat that are not fully grown.

![plz1](https://user-images.githubusercontent.com/53819958/97870905-ff25ea80-1d56-11eb-996b-0963b5b49337.PNG)
this is event(on break) code (i called function)
in conclusion, i would like to know the name of the plant.

### Server Information
* **Server version/platform:** 1.16.3 git-paper-231
* **Skript version:** 2.5.1

**Comment by ShaneBeee** [comment]
I can replicate this issue.

Marking this as a high priority because it is definitely a big bug.

**Comment by ElseJunsuk** [comment]
> I can replicate this issue.
> 
> Marking this as a high priority because it is definitely a big bug.

Okay, Thanks you

**Comment by iOshawott** [comment]
I don't know but maybe it's the same problem?
```
command /cotozaitem:
	trigger:
		send "%player's tool%"
```
it tells strange things for potions (every potion is luck potion) and the grass block is ... snowy grass?

**Comment by ShaneBeee** [comment]
Yeah the comparisons work (thank god) but the toString method for ItemTypes for some reason isn't working as it should.
